c10/core/SymIntArrayRef.h

```diff
@@ -79,6 +79,13 @@ class SymIntArrayRef final {
   /* implicit */ constexpr SymIntArrayRef(const c10::SymInt (&Arr)[N])
       : wrapped_symint_array_ref(Arr) {}
 
+  // This constructor is appropriate to use if you know that all the
+  // members of the IntArrayRef are positive
```

Comment:
1) positive or nonnegative?
2) might be nice to provide `fromIntArrayRefKnownNonnegative` (or positive, as appropriate) to self-document why it's unchecked at callsites

Reply:
whoops, non-negative